### PR TITLE
Clarify Example

### PR DESCRIPTION
**Pull request by DonRichards**
**GitHub Issue**: N/A

# What does this Pull Request do?
Changes the migration names and filenames associated to make it more clear as to what parts of the migration are referencing another migration vs a CDV cell vs a system function. 

# What's new?
Naming conventions

# How should this be tested?
Run a test migration and verify the names are correct and not misleading.

# Additional Notes:
N/A

# Interested parties
@Islandora/8-x-committers

### CODE REVIEW

File: config/install/migrate_plus.migration.example_migration_node.yml [hunk 1]
```diff
@@ -166,3 +166,8 @@ process:
 destination:
   plugin: 'entity:node'
   default_bundle: islandora_object
+
+migration_dependencies:
+  required:
+    - migrate_plus.migration.example_migration_file
```
Comment (seth-shaw-unlv) [inline config/install/migrate_plus.migration.example_migration_node.yml]: We don't need to make the node migration dependent on the file migration. It doesn't matter which of the two run first, just so long as they both run before the media one.

Comment (DonRichards) [reply config/install/migrate_plus.migration.example_migration_node.yml]: Good point.

Comment (DonRichards) [reply config/install/migrate_plus.migration.example_migration_node.yml]: @seth-shaw-unlv done. When debugging I controlling the firing make troubleshooting "seem" easier for me. lol. Anyways, it's ready for testing.

**Review by seth-shaw-unlv — APPROVED** [review]
👍

**Comment by seth-shaw-unlv** [comment]
@DonRichards, can you submit another PR fixing the migration IDs [in the documentation](https://github.com/Islandora/documentation/blob/master/docs/technical-documentation/migrate-csv.md)?

**Pull request merged**